corehq/apps/hqcase/tests/test_utils.py

```diff
@@ -112,3 +116,20 @@ def get_case(*args, **kwargs):
         yield case
     finally:
         factory.close_case(case.case_id)
+
+
+class TestIsCopiedCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.case_1 = create_case(domain=DOMAIN, name='case_1', case_json={'type': 'original'})
+        cls.case_2 = create_case(
+            domain=DOMAIN,
+            name='case_2',
+            case_json={'type': 'copied', CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME: cls.case_1.case_id}
+        )
+
+    def test_is_copied_case(self):
+        self.assertFalse(is_copied_case(self.case_1))
+        self.assertTrue(is_copied_case(self.case_2))
```

Comment:
>When possible, we'd like if the test body included all the context needed to determine what is going on. In this case, the reader doesn't know what makes case_1 different from case_2 without consulting the setUpClass method.

Great point. I have kept both the cases in `setupClass` method as `case_2` is supposed to be copy of `case_1`. It references case_1.case_id. As you suggested, this can be improved with the helper method,

Agreed on splitting the tests.